### PR TITLE
CR-1032457 Display warning if OS or Kernel isn't officially supported

### PR DESCRIPTION
**Pull request by AShivangi**

### CODE REVIEW

**Comment by maxzhen** [comment]
do we have a sample output?

**Comment by AShivangi** [comment]
When the kernel is not supported:
```
$ Debug/opt/xilinx/xrt/bin/xbutil scan
INFO: Found total 1 card(s), 1 are usable
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
System Configuration
OS name:        Linux
Release:        4.4.0-142-generic
Version:        #168-Ubuntu SMP Wed Jan 16 21:00:45 UTC 2019
Machine:        x86_64
Model:          Super Server
CPU cores:      16
Memory:         15715 MB
Glibc:          2.23
Distribution:   Ubuntu 16.04.5 LTS
Now:            Fri May  8 13:31:54 2020
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
XRT Information
Version:        2.6.0
Git Hash:       1d5eb0863db3e1cbfcdcd3bb6415bbb7b087ba9d
Git Branch:     xbutil-warning
Build Date:     2020-05-08 13:30:40
XOCL:           2.6.0,bd4d3bf50d763935d098fdcc0c108c7f5f171c0a
XCLMGMT:        2.6.0,bd4d3bf50d763935d098fdcc0c108c7f5f171c0a
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 [0] 0000:b3:00.1 xilinx_u200_xdma_201830_2(ID=0x5d1211e8) user(inst=128)
WARNING: Kernel version not supported.
```

To test the not supported case, I removed "4.4.0" from the list of supported versions

File: src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp [hunk 1]
```diff
@@ -89,6 +89,37 @@ static int str2index(const char *arg, unsigned& index)
...
+        }
+    }
+    else if(os.find("Red Hat") != std::string::npos || os.find("CentOS") != std::string::npos) {
+        for (const auto& ver : centos_rh_kernel_versions) {
+            if (release.find(ver) != std::string::npos) {
+                ret = true;
+                break;
+            }
+        }
+    }
```
Comment (maxzhen) [inline src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp:119]: can we get a function to get rid of the dup code?

Comment (AShivangi) [reply src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp]: done

**Pull request merged**